src/tutorial/packaging.md

```diff
@@ -1,7 +1,217 @@
+With `cargo publish`,
+you too can publish crates to [crates.io].
+And this works for all crates,
+including those with binary targets.
+
+Publishing a crate to [crates.io] is pretty straight-forward:
+If you haven't already, create an account on [crates.io].
+Currently, this is done via authorizing you on Github,
+so you'll need to have Github account
+(and be logged in there).
+Next, you need to log in with cargo.
+For that, go to your
+[crates.io account page],
+create a new token,
+and then run `cargo login <your-new-token>`.
+You only need to do this once per computer.
+You can learn more about this
+in cargo's [publishing guide].
+
+Now that cargo as well crates.io know you,
+you can publish crates.
+Before you hastily go ahead and publish a new crate version,
+it's a good idea to open your `Cargo.toml` once more
+and make sure you added a the necessary metadata.
+You can find all the possible fields you can set
+in the documentation for [cargo's manifest format].
+Here's a quick overview of some common entries:
+
+```toml
+[package]
+name = "grrs"
+version = "0.1.0"
+authors = ["Your Name <your@email.com>"]
+license = "MIT OR Apache-2.0"
+description = "A tool to search files"
+readme = "README.md"
+homepage = "https://github.com/you/grrs"
+repository = "https://github.com/you/grrs"
+keywords = ["cli", "search", "demo"]
+categories = ["command-line-utilities"]
+```
+
+<aside class="note">
+
+**Note:**
+This example includes the mandatory license field
+with a common choice for Rust projects:
+The same license that is also used for the compiler itself.
+It also refers to a `README.md` file.
+It should include a quick description of what your project is about,
+and will be included not only on the crates.io page of your crate,
+but also what Github shows by default on repository pages.
+
+</aside>
+
+[crates.io]: https://crates.io/
+[crates.io account page]: https://crates.io/me
+[publishing guide]: https://doc.rust-lang.org/1.30.0/cargo/reference/publishing.html
+[cargo's manifest format]: https://doc.rust-lang.org/1.30.0/cargo/reference/manifest.html
+
+### How to install a binary from crates.io
+
+Now that we've seen how to publish a crate to crates.io,
+you might be wondering how to install it.
+In contrast to libraries,
+that cargo will download and compile for you
+when you run `cargo build` (or a similar command),
+you'll need to tell it to explicitly install binaries.
+
+This is done using
+`cargo install <crate-name>`.
+It will by default download the crate,
+compile all the binary targets it contains
+(in "release" mode, so it might take a while)
+and copy them into the `~/.cargo/bin/` directory.
+(Make sure that your shell knows to looks there for binaries!)
+
+It's also possible to
+install crates from git repositories,
+only install specific binaries of a crate,
+and specify and alternative directory to install them to.
+Best have a look at `cargo install --help`.
```

Comment:
```suggestion
Best thing is to have a look at `cargo install --help`.
```

Reply:
I'm pretty sure "Best have a look at" is a thing people say. Might be colloqial, though. I'll try to come up with a completely different way to express this :)